seed/challenges/01-front-end-development-certification/html5-and-css.json

```diff
@@ -854,8 +851,7 @@
             "<blockquote>h1 {<br>&nbsp;&nbsp;font-size: 30px;<br>}</blockquote>",
             "Создайте второй элемент <code>p</code> со следующи текстом <code>kitty ipsum</code>: <code>Purr jump eat the grass rip the couch scratched sunbathe, shed everywhere rip the couch sleep in the sink fluffy fur catnip scratched.</code>",
             "Внутри того же элемента <code>&#60;style&#62;</code>, который содержит ваш класс <code>red-text</code>, создайте вхождение для элементов <code>p</code> и установите свойство <code>font-size</code> равным 16 пикселей (<code>16px</code>).",
-            "<strong>Внимание</strong><br>По причине разницы в реализации браузеров, вам может понадобиться установить 100% масштаб окна браузера для прохождения этого испытания.",
-            "Также, пожалуйста, не добавляйте атрибут класс к вашему новому элементу <code>p</code>."
+            "<strong>Внимание</strong><br> пожалуйста, не добавляйте атрибут класс к вашему новому элементу <code>p</code>."
```

Comment:
Capitalization of this is a bit more complicated, because I don't understand this alphabet (I do speak limited German and French). Google Translate tells me it's `П` instead of `п`. I'll try to consult my Russian-speaking friends about this one.